main.tf

```diff
@@ -349,6 +349,14 @@ resource "kubernetes_deployment" "this" {
           name              = "spotinst-kubernetes-cluster-controller"
           image_pull_policy = var.image_pull_policy
 
+
+          resources {
```

Comment:
BTW, this change makes it so that this module requires version 2.x of the Kubernetes provider: https://github.com/hashicorp/terraform-provider-kubernetes/issues/1169